examples/el_key/src/lib.rs

```diff
@@ -0,0 +1,446 @@
+//! An exemple that uses the `el_key` property of virtual DOM elements to define correspondence
+//! betweeen new and old elements.  CSS animation shows how elements are updated, added and
+//! removed.
```

Comment:
Could you write a simple guide how to use the example? I'm just clicking on some buttons and don't see any differences 😄 
Either as a comment here or maybe it would better directly in the HTML so users can follow the steps when they start the example.

E.g.
1. Uncheck `Use element keys`.
1. Click `A <-> B`.
1. Notice the wrong order of xxx and stopped animation in the card xxx....
.. etc, you got the idea.